### PR TITLE
Modified the team section to update the required bugs

### LINKED ISSUE #117: [BUG 🐞] <team section>
### Description

Here the main thing is, that after going to the team section, we can't redirect to the FAQ section directly, also it redirects to the homepage only whatever you do. I am opening this issue to debug this bug here.

### PR DESCRIPTION
**Pull request by alucard017**
Fixes #117 and #118
As mentioned in these issues, I have cleared all the bugs and updated everything.

### CODE REVIEW

File: team_list.html [hunk 1]
```diff
@@ -46,10 +46,10 @@
       <div class="right">
         <ul>
           <li class="about"><a  href="index.html#about">About</a></li>
-          <li class="services"><a target="_blank" href="https://github.com/orgs/OpenCodeEra/repositories">Projects</a></li>
+          <li class="services"><a href="index.html#projects">Projects</a></li>
           <li><a href="/team_list.html">Team</a></li>
           <li class="contact-us"><a  href="index.html#contact">Contact Us</a></li>
-          <li class="about"><a  href="index.html">FAQ</a></li>
+          <li class="about"><a  href="index.html#faq">FAQ</a></li>
```
Comment (Kamalkoranga) [inline team_list.html:52]: index.html#faq i.e. line 52 is not working !

**Review by Kamalkoranga — CHANGES_REQUESTED** [review]
Solve only one bug at a time.
Fix line 52 in `team_list.html` file i.e. link for faq in index file.

File: team_list.html (hunk 1 above)
Comment (alucard017) [reply team_list.html:52]:
```suggestion
          <li class="about"><a  href="index.html#faq">FAQ</a></li>
          It is perfectly working in my local environment. I think you should recheck it.
```

File: index.html
[the remark this answers is not in the document]
Comment (alucard017) [reply index.html:1]: Okay, I will keep track of the same from now on.

File: team_list.html (hunk 1 above)
Comment (alucard017) [reply team_list.html:52]: I have added the link to the #faq section present in <index.html> to the file <team_list.html>.

Comment (Kamalkoranga) [reply team_list.html:52]: Okay... It's working.
Push the code of `team_list.html` file only. So that I can merge it.

Comment (alucard017) [reply team_list.html:52]: I have pushed the code and revoked all the changes made to <index.html> file, also, updated <team_list.html> as required.

**Comment by AdarshRawat1** [comment]
LGTM @Kamalkoranga

**Review by Kamalkoranga — APPROVED** [review]

**Pull request merged**